azurelinuxagent/common/cgroupapi.py

```diff
@@ -241,17 +245,105 @@ class SystemdCgroupsApi(CGroupsApi):
     """
     Cgroups interface via systemd
     """
+
+    @staticmethod
+    def create_and_start_unit(unit_filename, unit_contents):
+        try:
+            fileutil.write_file("/etc/systemd/system/{0}".format(unit_filename), unit_contents)
```

Comment:
Can you make `/etc/systemd/system/` a variable as well? Tomorrow we might have a new location of systemd daemons, which we might want to handle.